.github/workflows/preview-env-build.yml

```diff
@@ -0,0 +1,71 @@
+---
+name: Preview Environment Build
+
+on:
+  pull_request:
+    types: [ labeled,synchronize ]
+
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}
+
+defaults:
+  run:
+    # use bash shell by default to ensure pipefail behavior is the default
+    # see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference
+    shell: bash
+
+jobs:
+  build-operare:
+    if: github.event.pull_request.state != 'closed' && (contains( github.event.label.name, 'deploy-preview') || contains( github.event.pull_request.labels.*.name, 'deploy-preview'))
+    name: Operate Build
+    uses: ./.github/workflows/operate-ci-build-reusable.yml
+    secrets: inherit
+    with:
+      branch: ${{ github.head_ref }}
+      previewEnv: true
+      commitSha: ${{ github.sha }}
+
+  build-tasklist:
+    if: github.event.pull_request.state != 'closed' && (contains( github.event.label.name, 'deploy-preview') || contains( github.event.pull_request.labels.*.name, 'deploy-preview'))
+    name: Tasklist Build
+    uses: ./.github/workflows/tasklist-ci-build-reusable.yml
+    secrets: inherit
+    with:
+      branch: ${{ github.head_ref }}
+      previewEnv: true
+      commitSha: ${{ github.sha }}
+
+  build-zeebe:
+    if: github.event.pull_request.state != 'closed' && (contains( github.event.label.name, 'deploy-preview') || contains( github.event.pull_request.labels.*.name, 'deploy-preview'))
+    name: Zeebe Build
+    runs-on: ubuntu-latest
+    concurrency:
+      group: deploy-docker-snapshot
+      cancel-in-progress: false
```

Comment:
I think this `concurrency` setting on the job level needs to be removed since it is redundant since this job is not pushing the `SNAPSHOT` Docker image and will block other real `deploy-docker-snapshot` jobs.

Reply:
ah yeah, good point!